internal/attestation/measurements/measurements_test.go

```diff
@@ -321,3 +321,95 @@ func TestMeasurementsFetchAndVerify(t *testing.T) {
 		})
 	}
 }
+
+func TestAllBytes(t *testing.T) {
+	testCases := map[string]struct {
+		b       byte
+		wantPCR []byte
+	}{
+		"0x00": {
+			b:       0x00,
+			wantPCR: []byte{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+		},
+		"0x01": {
+			b:       0x01,
+			wantPCR: []byte{0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01},
+		},
+		"0xFF": {
+			b:       0xFF,
+			wantPCR: []byte{0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF},
+		},
+	}
+
+	for name, tc := range testCases {
+		t.Run(name, func(t *testing.T) {
+			assert := assert.New(t)
+			pcr := AllBytes(tc.b)
+			assert.Equal(tc.wantPCR, pcr)
+		})
+	}
+}
+
+func TestZero(t *testing.T) {
+	zero := []byte{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00}
+	pcr := Zero()
+	assert.Equal(t, zero, pcr)
+}
+
+func TestOne(t *testing.T) {
+	one := []byte{0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF}
+	pcr := One()
+	assert.Equal(t, one, pcr)
+}
+
+func TestMeasurements_EqualTo(t *testing.T) {
```

Comment:
nit: not sure what our naming convention is for this kind of tests. @katexochen

Reply:
In most cases we would just have called this `TestEqualTo` I guess, but for me its okay this way, too, as github.com/cweill/gotests (included in the VSCode Go ext) uses this naming convention. Maybe something we could start to adopt...